pkg/webreg/webreg.go

```diff
@@ -1880,37 +1879,310 @@ as the component.
+teams. After a PR is filed, the bot assigns two reviewers who should be suitable
+to review the PR and are expected to do so. These people are also the ones to
+bug when a PR sits there without a review. Teams are expected to own their CI
+config, including reviews, and therefore <code>OWNERS</code> file presence is
+enforced for some sections of the repository.
+</p>
+
+<p>
+During the PR lifetime, the bot maintains a comment that summarizes the approval
+status of the pull request, including the links to the <code>OWNERS</code> files
+whose members need to approve the PR. Please pay attention to this comment when
+asking for approvals.
+<!--TODO: a screenshot would be nice?-->
+</p>
+
+<p>
+Due to the pull request volume in the repository, the members of the DPTP team
+are reviewing the pull requests asynchronously when assigned by a bot. Please do
+not expect a PR to be reviewed immediately. Unless really urgent, do not ping
+about reviews via Slack. If a PR sits unreviewed for more than a day, ping via
+GitHub first via a mention. If a pull request spends some time in WIP or draft
+state, it is helpful to mention the reviewers when the PR is ready for review.
+</p>
+
+<h4 id="checks"><a href="#checks">Checks</a></h4>
+
+<h5 id="formatting-checks"><a href="#formatting-checks">Formatting and Generated Content</a></h5>
+
+<p>
+Parts of the repository content are partially or entirely managed by automation
+and there are checks in place enforcing that the repo stays consistent w.r.t.
+all this automation. When these checks fail, they usually advise how to run the
+tooling (using containers) to bring the repo to the desired state:
+</p>
+
+{{ plaintextSyntax (index . "determinizeCheckExample") }}
+
+<p>
+While there are individual <code>make</code> targets for different parts of the
+repository, it is easiest to run the <code>make update</code> that runs <em>all</em>
+these tools before a pull request submission:
+</p>
+
+{{ plaintextSyntax (index . "makeUpdateExample") }}
+
+<h5 id="rehearsals"><a href="#rehearsals">Rehearsals</a></h5>
+
+<p>
+In addition to the "normal" checks executed against pull requests on <code>openshift/release</code>,
+so-called <em>"rehearsals"</em> are triggered whenever a pull request would
+affect one or more CI jobs. Jobs that would be affected by such PR are executed
+as if run against a target component repository after the changes would be
+merged. This provides pull request authors early feedback about how config
+changes impact CI setup.
+</p>
+
+<p>
+All pull requests trigger a <code>ci/prow/pj-rehearse</code> job which inspects
+the changes in the PR and detects affected jobs. It then submits these jobs for
+execution and they will report to the pull request results via the GitHub
+contexts named with the <code>ci/rehearse/$org/$repo/$branch/$test</code>
+pattern. Neither the "driver" job (<code>ci/prow/pj-rehearse</code>) nor the
+individual rehearsals do not block merges.
```

Comment:
```suggestion
individual rehearsals are optional, so they may be ignored when merging configuration.
```

Reply:
Maybe also say why you might want to?

Reply:
I'd like to keep the simple sentence here ("do not block merges") but explaining why is useful so I'll add that.